app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -171,20 +171,30 @@ function buildArticleHTML(article) {
     var isArticle = article.class_name === 'Article';
 
     var previewCardContent = `
-      <div id="story-author-preview-content-${article.id}" class="profile-preview-card__content crayons-dropdown p-4" data-repositioning-dropdown="true" style="border-top: var(--su-7) solid var(--card-color);" data-testid="profile-preview-card">
+      <div id="story-author-preview-content-${
```

Comment:
This line breaking appears to be part of the pre-commit hooks.  I swear I didn't do this.